nstars.c

```diff
@@ -157,7 +157,7 @@ int main(int argc, char *argv[])
 					beep();
 				}
 				break;
-			case 10:					/* Enter */
+			case KEY_ENTER:
```

Comment:
This didn't work in my case. From ncurses getch manual page:

```
       KEY_ENTER  versus  control/M, KEY_BACKSPACE versus control/H.  Some curses
       implementations may differ according to whether they treat  these  control
       keys specially (and ignore the terminfo), or use the terminfo definitions.
       Ncurses uses the terminfo definition.  If it says that KEY_ENTER  is  con‐
       trol/M, getch will return KEY_ENTER when you press control/M.

       Generally, KEY_ENTER denotes the character(s) sent by the Enter key on the
       numeric keypad:

       ·   the terminal description lists the most useful keys,

       ·   the Enter key on the regular keyboard is already handled by the  stan‐
           dard ASCII characters for carriage-return and line-feed,

       ·   depending  on  whether  nl or nonl was called, pressing "Enter" on the
           regular keyboard may return either a carriage-return or line-feed, and
           finally

       ·   "Enter or send" is the standard description for this key.

```

So, it sound like the best way to handle all possibilities is:

```
diff --git a/nstars.c b/nstars.c
index 353ee4f..7037850 100644
--- a/nstars.c
+++ b/nstars.c
@@ -157,6 +157,8 @@ int main(int argc, char *argv[])
                                        beep();
                                }
                                break;
+                       case 10:                                        /* Line Feed */
+                       case 13:                                        /* Carriage Return */
                        case KEY_ENTER:
                                if (strncmp(cmd_buf_p, "/quit", MAX_CMD_BUF_LEN) == 0) {
                                        quit = 1;
```